python_modules/libraries/dagster-databricks/README.md

```diff
@@ -3,36 +3,38 @@
 The docs for `dagster-databricks` can be found
 [here](https://docs.dagster.io/_apidocs/libraries/dagster-databricks).
 
-## ext example
+## Pipes example
 
-This package includes a prototype API for launching databricks jobs with
-Dagster's EXT protocol. There are two ways to use the API:
+This package includes a prototype API for launching Databricks jobs with
+Dagster's Pipes protocol. There are two ways to use the API:
 
-### (1) `ExtDatabricks` resource
+### (1) `PipesDatabricksClient` resource
 
-The `ExtDatabricks` resource provides a high-level API for launching
-databricks jobs using Dagster's ext protocol.
+The `PipesDatabricksClient` resource provides a high-level API for launching
+Databricks jobs using Dagster's Pipes protocol.
 
-`ExtDatabricks.run` takes a single `databricks.sdk.service.jobs.SubmitTask`
-specification. After setting up ext communications channels (which by default
-use DBFS), it injects the information needed to connect to these channels from
-Databricks into the task specification. It then launches a Databricks job by
-passing the specification to `WorkspaceClient.jobs.submit`. It polls the job,
-state, exits gracefully on success or failure, and returns a stream of
-`MaterializeResult` events that should be yielded:
+`PipesDatabricksClient.run` takes a single
+`databricks.sdk.service.jobs.SubmitTask` specification. After setting up Pipes
+communications channels (which by default use DBFS), it injects the information
+needed to connect to these channels from Databricks into the task
+specification. It then launches a Databricks job by passing the specification
+to `WorkspaceClient.jobs.submit`. It synchronously executes the job and returns
+a `PipesClientCompletedInvocation` object that exposes a `get_results` method.
+The output of `get_results` is a tuple of `PipesExecutionResult` objects that
+you can return from the asset compute function.
 
 
 ```
 import os
 from dagster import AssetExecutionContext, Definitions, asset
-from dagster_databricks import ExtDatabricks
+from dagster_databricks import PipesDatabricksClient
 from databricks.sdk import WorkspaceClient
 from databricks.sdk.service import jobs
 
 @asset
-def databricks_asset(context: AssetExecutionContext, ext: ExtDatabricks):
+def databricks_asset(context: AssetExecutionContext, pipes_client: PipesDatabricksClient):
```

Comment:
Are we sure `pipes_client` is the naming convention we want? People will copy this.

`pipes`
`pipes_databricks_client`
`databricks_client`
`databricks_pipes`

I guess would be other options?

Reply:
IMO `pipes_client` is the best balance of clarity and concision.